src/main/java/net/minecraftforge/common/extensions/IForgeJukebox.java

```diff
@@ -0,0 +1,23 @@
+package net.minecraftforge.common.extensions;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.LevelAccessor;
+import net.minecraft.world.level.block.JukeboxBlock;
+import net.minecraft.world.level.block.state.BlockState;
+
+import javax.annotation.Nullable;
+
+public interface IForgeJukebox
```

Comment:
Since this is not a standard extension method, the interface should be documented itself.

Reply:
Added